pkg/printer/printer.go

```diff
@@ -13,7 +13,10 @@ import (
 
 // Printer is an interface for printing FileResults
 type Printer interface {
-	Print(io.Writer, *result.FileResults) error
+	Print(*result.FileResults) error
+	Start()
+	End()
+	ShouldSkipExitMessage() bool
```

Comment:
Reversed the naming of this to `PrintSuccessExitMessage( )`.  Good question about the success error message feature. I was thinking more generically about adding a "output" section of the configuration for each output type. Then success message would make sense, along with options such as the ability to change the severity mappings for sonarqube and github-actions, ability to choose single json object vs list of json objects for json output type, "effortMinutes" for sonarqube (currently I don't have this included since it would take an option) etc. Sorry this response is getting off-topic :) let me put in a Feature Request so we can discuss there.